syntax = "proto3";

package payload;

message GCS_TO_AV_STATE_CMD {
    optional bool main_secondary_test = 1;
    optional bool main_primary_test = 2;
    optional bool apogee_secondary_test = 3;
    optional bool apogee_primary_test = 4;

    optional bool main_secondary_test_inverted = 5;
    optional bool main_primary_test_inverted = 6;
    optional bool apogee_secondary_test_inverted = 7;
    optional bool apogee_primary_test_inverted = 8;

    optional bool broadcast_begin = 9;
}